sources/operator.1.14.0.psmdb-operator.json

```diff
@@ -0,0 +1,156 @@
+{
+  "versions": [
+    {
+      "operator": "1.14.0",
+      "product": "psmdb-operator",
+      "matrix": {
+        "mongod": {
+          "6.0": {
+            "image_path": "percona/percona-server-mongodb:6.0",
+            "image_hash": "947f2557787e20b36345d5aa9c9ca42bbc00f3e4d97a445e46ebb2cf8e49def8",
+            "status": "recommended",
+            "critical": false
+          },
+          "5.0.7-6": {
+            "image_path": "percona/percona-server-mongodb:5.0.7-6",
+            "image_hash": "3f4849a17236c3849a513f46caa39fbc6da0414f98d27e080fbe0496fa9e86a2",
+            "status": "available",
```

Comment:
One version from each major version needs to be tagged as `recommended` (so one 6.0 needs to be recommended, one 5.0, one 4.4, etc.).
Also we support tags like `5.0-recommended, `5.0-latest` and we need to have that support for 6.0 too.

Reply:
Tests need to be updated also.

Reply:
Done